src/Avalonia.Base/Input/KeyboardNavigationHandler.cs

```diff
@@ -113,6 +117,20 @@ void OnKeyDown(object? sender, KeyEventArgs e)
                 Move(current, direction, e.KeyModifiers);
                 e.Handled = true;
             }
+            else if (e.Key is Key.Left or Key.Right or Key.Up or Key.Down)
```

Comment:
There are no unit tests covering this block of code: might be worth adding one or two just to exercise this code path?

Reply:
Yeah, there should be. And this code is a reason why Menu tests failed, haha.